### ISSUE TITLE
The request is not allowed by the user agent or the platform in the current context.

### ISSUE THREAD

**Issue by pannous** [ScaleDrone/webrtc#3]
demo https://scaledrone.github.io/webrtc/index.html#bda2af silently fails. console:
```
onaddstream is deprecated! Use peerConnection.ontrack instead.
script.js:72
MediaStreamError { name: "NotAllowedError", message: "The request is not allowed by the user agent or the platform in the current context.", constraint: "", stack: "" }
script.js:22:3
```

**Comment by herkyl** [comment]
Will patch this soon.

**Comment by herkyl** [comment]
Fixed with 650a2da2ceb96b7f9a312ecaf5bad609de478f4d
WebRTC changes are still causing some problems for setting up a reliable connection though.
